internal/gather/connectivity.go

```diff
@@ -32,6 +32,7 @@ const (
 	addonPodLabel            = "app=submariner-addon"
 	ovnMasterPodLabelOCP     = "app=ovnkube-master"
 	ovnMasterPodLabelGeneric = "name=ovnkube-master"
+	ovnKubePodLabel          = "app=ovnkube-node"
```

Comment:
The ovnkube-node pod has multiple containers in it and not all of them include the ovn-nbctl and ovn-sbctl commands. We should also pass the container name like nbdb/sbdb while collecting the logs to have a deterministic behavior.

List of containers: ovn-controller, ovn-acl-logging, kube-rbac-proxy-node, kube-rbac-proxy-ovn-metrics, northd, nbdb, sbdb, ovnkube-controller

```
bash-4.4$ ovn-nbctl --no-leader-only list Logical_Router
bash: ovn-nbctl: command not found
bash-4.4$
```

Reply:
`logCmdOutput` API uses `container[0]` by default and `ovnkube-node` DaemonSet defines `ovn-controller` at [0], so it should be deterministic.

We will have to refactor the API to allow caller to specify container name and that refactor is beyond scope of this change.

Reply:
I'm aware that we currently rely on `container[0]` in our code, but given the ongoing changes in the OVN side due to IC, it's advisable to refrain from making any assumptions.

If it requires major changes, lets not forget to do it in a follow on PR.